gitops/git/git.go

```diff
@@ -103,6 +104,34 @@ func (r *Repo) Commit(message, gitopsPath string) bool {
 	return true
 }
 
+// RemoveDiff removes the changes made to a specific file in the repository
+func (r *Repo) RemoveDiff(fileName string) {
+	exec.Mustex(r.Dir, "git", "checkout", "--", fileName)
+}
+
+// split by newline and ignore empty strings
+func SplitFunc(c rune) bool {
+	return c == '\n'
+}
+
+// GetChangedFiles returns a list of files that have been changed in the repository
+func (r *Repo) GetChangedFiles() []string {
+	files, err := exec.Ex(r.Dir, "git", "diff", "--name-only")
+	if err != nil {
+		log.Fatalf("ERROR: %s", err)
+	}
+	return strings.FieldsFunc(files, SplitFunc)
```

Comment:
I'd prefer the use of the Scanner.readLine(). While it will be more verbose it will be more obvious what empty lines re not returned. 
```go
    var files []string
    sc := bufio.NewScanner(strings.NewReader(s))
    for sc.Scan() {
        files = append(files, sc.Text())
    }
    return lines
```